File: Solo_grupp/API.Data/Models/News.cs
namespace API.Data.Models
{
	#region Using
	using System;
	using System.Collections.Generic;
	#endregion
	public class News
	{
		public Guid Id { get; set; }
		public string Title { get; set; }
		public string Content { get; set; }
		public DateTime CreateDate { get; set; }
		public virtual User User { get; set; }
		public virtual ICollection<Image> Images { get; set; }
		public virtual ICollection<Comment> Comments { get; set; }

		public News()
		{
			this.Id = Guid.NewGuid();
		}
	}
}

File: Solo_grupp/API/Scripts/Common/Models/News.ts
import {User} from './User.ts';
import {Comment} from './Comment.ts';

export class News {
    public Id: string = null;
    public Content: string = null;
    public Title: string = null;
    public Author: User = null;
    public CreateDate: Date = null;
    public Comments: Comment[] = null;
}